backend/src/common/utility/validation.utils.ts

```diff
@@ -0,0 +1,7 @@
+export class ValidataionUtils {
+    static isString(val: any | undefined | null) {
```

Comment:
Tau tikrai reikia runtime type checkerio?

Reply:
aš buvau vienu momentu pradėjęs rankom validuot, bet tada perėjau prie `class-validator`. Šitą pašalinsiu, thanks